quodlibet/cli.py

```diff
@@ -239,6 +241,13 @@ def queue(*args):
             queue(command, filename)
         elif command == "enqueue-files":
             queue(command, arg)
+        elif command == "set-pattern":
+            try:
+                queue(command, args[0])
```

Comment:
I think this should just be `arg`. This will fail if `--set-pattern` isn't the first option provided.